test/sources/npm_test.rb

```diff
@@ -80,5 +80,27 @@
         end
       end
     end
+
+    describe "missing dependencies (glob is missing package)" do
+      let(:fixtures) { File.expand_path("../../fixtures/npm/missing-dependencies", __FILE__) }
+
+      it "includes missing dependencies when yarn.lock is present" do
+        Dir.chdir fixtures do
+          File.unlink("yarn.lock") if File.exist? "yarn.lock"
```

Comment:
This test shouldn't need to clean up after the other test 👍 
```suggestion
```

Reply:
N/A after test rework.